src/main/java/org/tron/common/runtime/RuntimeImpl.java

```diff
@@ -346,20 +403,29 @@ private void create()
 
       AccountCapsule creator = this.deposit
           .getAccount(newSmartContract.getOriginAddress().toByteArray());
+
+      if (RuntimeImpl.isNewVersion()) {  // TODO according to version
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: Change this condition so that it does not always evaluate to "false" ([squid:S2583](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS2583))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1670&issues=AWbSdHUuSgN4uCclEWnD&open=AWbSdHUuSgN4uCclEWnD)

[ISSUE_KEY]: AWbSdHUuSgN4uCclEWnD